pages/shoppingCart/shoppingCartPage.ts

```
@@ -1,13 +1,58 @@
   constructor(page: Page) {
     super(page);
+    this.shoppingCartTable = this.page.locator('[class="table table-hover"]');
+    this.tableRaw = this.page.locator("tbody>tr");
+    this.productTitleCell = this.page.locator('[class="product-title"]');
+    this.headers = this.page.locator("thead");
   }
 
-  // Locators
-
   // Methods
+  async getShoppingCartTableContent() {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)